### DISCUSSION TITLE
How is this linked to Laravel?

### DISCUSSION THREAD

**Discussion by goldingdamien** [spatie/laravel-sitemap#395] [Q&A] [answered]
Does this need Laravel installed?
If it does, can this be run on any site by installing Laravel but not using Laravel for the actual site being crawled?
If so, how should Laravel be setup?

Regards.

**Comment by ousid** [comment, marked as answer]
Hey, @goldingdamien 
As far as I know The answer Yes and No, you can use the package without Laravel in condition (not an order) to fork the repository and make some modifications on the `composer.json` file such as removing the Laravel auto discovery feature, the artisan commands... and keep the required packages that the `sitemap` package depends on them to do the job.

Just to note: The package depends on `orchestra/testbench` and it's Laravel Testing Helper for Packages Development.

You got what you need and the decision is yours :)
